vars/pssh.groovy

```diff
@@ -2,12 +2,27 @@
 
 void call(Set<String> hosts, String command) {
   sh """
-    pssh \
-      -H "${hosts.sort().join(' ')}" \
-      -O StrictHostKeyChecking=no \
-      -O UserKnownHostsFile=/dev/null \
-      -t 300 \
-      --inline \
-      "${command}"
+    hosts="${hosts.sort().join(' ')}"
+    tmpdir=/tmp/pssh.\$\$
+    mkdir -p \$tmpdir
```

Comment:
[mktemp](https://linux.die.net/man/1/mktemp) would probably be a better fit here as it would avoid collisions if this gets called multiple times simultaneously

Reply:
good call!